skiko/src/jvmTest/kotlin/org/jetbrains/skiko/BitmapTest.kt

```diff
@@ -0,0 +1,46 @@
+package org.jetbrains.skiko
```

Comment:
Shalln't it be in just `org.jetbrains.skia` package? Would simplify imports a lot. Not blocking, ofc.